styles/AdvancedEditor.css

```diff
@@ -558,73 +554,60 @@ Next levels are shifted 'up' by 1
     margin-top: 4px !important;
 }
 
-.keywords-list {
-  margin: 0;
-  display: inline;
-}
-
-.keywords-list span.keyword {
+.keywords .keyword {
   display: inline-block;
   color: #353535;
   background-color: #f2f2f2;
-  font-family: Lato, sans-serif;
+  font-family: Lato,sans-serif;
   font-size: 14px;
   line-height: 16px;
   margin-right: 4px;
   margin-bottom: 4px;
-  border: 1px solid #f2f2f2;
+  border: 1px solid transparent;
   border-radius: 6px;
   position: relative;
   cursor: default;
+  padding: 4px 8px
 }
 
-.keywords-list span.keyword.no-tracking {
-  padding: 4px 8px;
-}
-
-.keywords-list span.keyword ins,
-.keywords-list span.keyword del {
-  display: inline-block;
-  padding: 4px 8px;
-  border-radius: 6px;
+.keywords .keyword ins, .keywords .keyword del {
+  background-color: inherit !important
 }
 
-.keywords-list span.keyword:hover,
-.keywords-list span.keyword:hover ins,
-.keywords-list span.keyword:hover del {
-  background-color: #f2fbfc;
-  border-color: #f2fbfc;
+.keywords .keyword:hover {
+  background-color:#f2fbfc !important;
 }
 
-.keywords-list span.keyword.selected {
+.keywords .keyword.selected {
   background-color: #f2fbfc;
   border-color: #bce7f6;
 }
 
-.keywords-list .keyword.deleted.pending,
-.keywords-list .keyword.deleted.accepted {
+.keywords .keyword.deleted.pending,.keywords .keyword.deleted.accepted {
   text-decoration: line-through;
 }
-.keywords-list .keyword .track-changes--control {
+
+.keywords .keyword .track-changes--control {
   top: 0;
 }
 
-.keywords-list span.delete-keyword {
+.keywords .delete-keyword {
   cursor: pointer;
   margin: 0 0 0 8px;
-  opacity: 0.5;
+  opacity: .5;
 }
 
-.keywords-list span.delete-keyword:hover {
+.keywords .delete-keyword:hover {
   opacity: 1;
 }
 
-.keywords-list .deleted.accepted span.delete-keyword {
+.keywords .deleted .delete-keyword {
```

Comment:
we need to add `.accepted`

Reply:
Why is that needed?

Reply:
to show delete button in case it was rejected